spec/unit/defines/server/dbgroup_spec.rb

```diff
@@ -51,7 +50,7 @@
   
     let :params do
       {
-        :groupmembers => ['testuser1', 'testuser2'],
+        :groupmembers => "{\"testuser1\", \"testuser2\"}",
```

Comment:
I much preferred the prior API. Is it possible to expose the previous API to the caller (array of names) and then do the escaping / formatting yourself inside `postgresql::server::dbgroup`?